randomStyle.js

```diff
@@ -0,0 +1,17 @@
+
+const h =document.querySelector('.height').value;
+h.setAttribute('value','height');
+
+const w =document.querySelector('.width').value;
+w.setAttribute('value','width');
+
+document.querySelector('#randomize').addEventListener('click', createImage);
```

Comment:
always put your event listener at the end of your code.
I mean after you declare the function.